python/TestHarness/tests/resultsstore/common.py

```diff
@@ -46,6 +46,11 @@ def random_id() -> int:
     return random.randint(1, 1000000)
 
 
+def random_version() -> str:
+    """Generate a random version."""
+    return random.randint(1, 9)
+
```

Comment:
Let's not do anything based on version in the summary tests (which means this function isn't needed). The testing on version should be in the results reader. The test in the summary should just be based on whether or not the memory value is none or not.